Makefile

```diff
@@ -137,4 +139,17 @@ asup:
 
 dev: build
 	@echo "Deleting AutoSupport binary"
-	@rm -rf autosupport/asup
\ No newline at end of file
+	@rm -rf autosupport/asup
+
+fetch-asup:
+	@mkdir -p ${ASUP_PAYLOAD_DIR}
+	@GOMODCACHE=${ASUP_PAYLOAD_DIR} go mod download github.com/rahulguptajss/Test@$(ASUP_BINARY_VERSION)
+	@$(eval MY_VAR=$(shell find ${ASUP_PAYLOAD_DIR} -name "*asup*" -name "*"$(GOOS)"*" -exec ls -t {} + | head -n 1))
+ifeq (${MY_VAR}, )
+	@\cp -rf $(MY_VAR) ${CURRENT_DIR}/autosupport/asup
+	@chmod u=rwx,g=rx,o=rx ${CURRENT_DIR}/autosupport/asup
```

Comment:
Due to `https://github.com/golang/go/issues/27455` might be worth adding this line so these directories are easier to cleanup
```
chmod -R +rw asup/github.com/
```